daml-lf/validation/src/main/scala/com/digitalasset/daml/lf/validation/Recursion.scala

```diff
@@ -25,6 +25,8 @@ private[validation] object Recursion {
     val modRefsInType: Set[ModuleName] = {
 
       def modRefsInType(acc: Set[ModuleName], typ0: Type): Set[ModuleName] = typ0 match {
+        case TSynApp(typeSynName,_) if typeSynName.packageId == pkgId =>
+          acc + typeSynName.qualifiedName.module
```

Comment:
```suggestion
          TypeTraversable(typ0).fodLeft(acc + typeSynName.qualifiedName.module)(modRefsInType)
```

Reply:
Thank you. I made the fix using `/:` to be consistent.

Reply:
[FYI, `/:` and `:\` are deprecated in favor of `foldLeft` and `foldRight` respectively.](https://www.scala-lang.org/api/2.12.10/scala/collection/TraversableOnce.html)

Reply:
All those `/:` are mine :( 
I will spend some time soon to remove all `/:`. 
You can use here whatever you prefer.